8_distributed_training.ipynb

```diff
@@ -33,7 +33,7 @@
    "source": [
     "### Preparing the data\n",
     "\n",
-    "Normally data would already be in a shared location, but since it's local, we'll upload it to our object storage so we can show sharded data loading from a shared data source. Once it's uploaded, we can work with it using Ray Data so it's properly sharded across the workers."
```

Comment:
> Ray Data so that it is properly shared across the worker machines

The second reference was intended to be **sharded**, not shared.  Maybe "split" is clearer?